.appveyor.yml

```diff
@@ -377,6 +377,15 @@ after_test:
         throw "Build should be up-to-date but isn't."
       }
 
+  # Verify that javascript and typescript files which are bundled by rollup are
+  # listed explicitly in BUILD.gn.
+  - ps: |-
+      Get-ChildItem "js" -File -Force -Name                          |
+        where   { -not $_ -like "*_test.ts" }                        |
+        where   { -not (Select-String -Pattern $_ -Path BUILD.gn `
+                                      -SimpleMatch -CaseSensitive) } |
+        foreach { throw "$_ should be listed in BUILD.gn but isn't." }
```

Comment:
This is brittle - if we rename js/ to ts/ for example this will break and it will be very silent. I'd rather not add such things. Better would be if rollup took an explicit list of inputs in the command-line - and thus wouldn't compile correctly if one was missing.

Reply:
The current situation is more brittle.

Reply:
If we rename js to ts then `Get-ChildItem js` would throw.

Reply:
I added a comment for your appeasement: https://github.com/denoland/deno/pull/738/files#diff-11c909939117928998b102a1fff7d363R381